### DISCUSSION TITLE
Hi, I would like used this browser in Adobe AIR

### DISCUSSION THREAD

**Discussion by eduardo12fox** [Mzying2001/CefFlashBrowser#49] [General]
Hello, I´m from Brazil and I would like put this browser into Adobe AIR sdk 4.6 you have or know how put this in adobe air?

**Comment by Mzying2001** [comment]
Hello, I'm not familiar with Adobe AIR, so I may not be able to provide support in this regard. This browser is based on development using WPF, which is used to support Flash content on webpages and can only run on Windows systems.
